src/core/display/Container.js

```diff
@@ -544,3 +544,12 @@ Container.prototype.renderCanvas = function (renderer)
         renderer.maskManager.popMask(renderer);
     }
 };
+
+/**
+ * Destroys the container
+ *
+ */
+Container.prototype.destroy = function ()
+{
+    this.children = null;
```

Comment:
It should definitely call the destroy method of children, and probably remove them as well.